release_notes/Service_Fabric_ReleaseNotes_90.md

```diff
@@ -0,0 +1,153 @@
+| 16. | **Feature** | Added ability to enumerate actor reminders | **Brief Description**: Exposed an API to list all the reminders in the actor service.GetReminderAsync() API has been added to IActorService to enable users to enumerate reminders. <br>**Solution**: Actor customers now has the ability to enumerate the reminders in their service. If a null value is passed to the actorId filter, then the Api returns list of all the reminders. If a valid actor id is passed, then all the reminders corresponding to the actor id input are returned.<table><tr><td><pre lang="csharp">Task<ReminderPagedResult<KeyValuePair<ActorId, List<ActorReminderState>>>> GetRemindersAsync(<br>    ActorId actorId,<br>    ContinuationToken continuationToken,<br>    CancellationToken cancellationToken);</pre></td></tr><tr><td></table>The page size of the result can be controlled by setting the PageSize static field of the ReminderPagedResult inside the actor service.<table><tr><td><pre lang="csharp">ReminderPagedResult<KeyValuePair<ActorId, List<ActorReminderState>>>.SetDefaultPageSize(/*new size*/)</pre></td></tr></table>**Documentation Reference**: Read more about it at -<ul><li>[IActorService.GetReminderAsync(ActorId, ContinuationToken, CancellationToken) Method](https://docs.microsoft.com/en-us/dotnet/api/microsoft.servicefabric.actors.iactorservice.getreminderasync)</li></ul> |
+| 17. | **Feature** | Made updates to MR UI | **Brief Description**: This update adds new page under each Infrastructure Service section which displays information related to undergoing Infrastructure updates on the Virtual Machine Scale Sets. Users can get information about currently Executing updates, Pending updates and the State of each update right from the Service fabric Explorer.<br>**Impact**: This helps in quickly identifying the undergoing infrastructure updates and understanding overall health of the Cluster.<br>Correlating Infrastructure updates with corresponding repair Tasks , to identify why the updates are stuck.<br>**Documentation Reference**: Read more about it at -<ul><li>[Introduction to the Service Fabric Infrastructure Service](https://docs.microsoft.com/en-us/azure/service-fabric/infrastructure-service-faq)</li><li>[Azure Service Fabric node types and virtual machine scale sets](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-nodetypes)</li></ul> |
```

Comment:
We don't expose MR externally -> I think this needs to be referred to as platform events.